source/glue/library/read_write.py

```diff
@@ -105,14 +107,26 @@ def load_input_data(self) -> None:
         for field in self.pii_fields:
             pii_column_names[field["column_name"]] = str
 
-        if self.content_type == JSON_CONTENT_TYPE:
+        content_type = self.content_type
+
+        # if a gzip, determine whether it is a zipped json or csv
+        if self.content_type == GZIP_CONTENT_TYPE:
+            # regex file name to see if json.gzip
```

Comment:
type-o in this comment. I think you want to say "json.gz" instead of "json.gzip"